LICENSE.rst

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
+
+Copyright (c) 2022-2024, The Trustees of Columbia University
```

Comment:
I changed to 2022-2024 based on the first commit being Feb 2022

https://github.com/diffpy/diffpy.nmf_mapping/commit/d40b80700d907648b2622cfa240bda7bfb8d0182